activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb

```diff
@@ -111,7 +111,7 @@ def #{sym}=(obj)
   #   class Customer < Account
   #   end
   #
-  #   Customer.user = "CHH"
+  #   Customer.user = "DHH"
```

Comment:
This is meant to demonstrate that a change in Customer won't affect Account, which this change breaks. I was confused first, so we should find a better example 😁